Automattic/wp-calypso
# Settings: highlight credentials settings section when redirected from the site activity

#### Changes proposed in this Pull Request

Right now we're sending customers from the site activity to security settings in case they haven't added server credentials yet. In these scenarios, it's easy to lose focus when jumping into settings. This PR helps focusing on the section that needs attention.

* Highlights credentials section in settings when sent from the site activity to help maintain context.
* Builds up on https://github.com/Automattic/wp-calypso/pull/40014

Note for reviewers: this implementation was done as the smallest MVP possible. I'm aware it is very fragile, so feel free to dismiss it.

![jp-credentials-highlight-v2](https://user-images.githubusercontent.com/390760/76335230-2291b600-62ec-11ea-977b-bf227f16f3f9.gif)


#### Testing instructions

* Fire up this PR locally or using jurassic.ninja.
* Connect Jetpack.
* Go to your site activity in Calypso; e.g.: `http://calypso.localhost:3000/activity-log/SITE_URL`.
* Click the top banner or the popover menu item to add server credentials.
* Ensure the correct section is highlighted when redirected to settings.
